pandas/tests/io/formats/style/test_style.py

```diff
@@ -313,26 +317,47 @@ def test_index_name(self):
                     "value": "B",
                     "display_value": "B",
                     "is_visible": True,
+                    "attributes": "",
                 },
                 {
                     "class": "col_heading level0 col1",
                     "type": "th",
                     "value": "C",
                     "display_value": "C",
                     "is_visible": True,
+                    "attributes": "",
                 },
             ],
             [
-                {"class": "index_name level0", "type": "th", "value": "A"},
-                {"class": "blank col0", "type": "th", "value": self.blank_value},
-                {"class": "blank col1", "type": "th", "value": self.blank_value},
+                {
+                    "class": "index_name level0",
+                    "type": "th",
+                    "value": "A",
+                    "is_visible": True,
+                    "display_value": "A",
+                },
+                {
+                    "class": "blank col0",
+                    "type": "th",
+                    "value": self.blank_value,
+                    "is_visible": True,
```

Comment:
so these are just additional attributes that currently are defaulted?

Reply:
yes the new `_element` function outputs 5 dict keys by default for each cell. 

Previously almost all cells needed these 5 and a few only had 3 keys like column header cells. There is no harm adding them, and actually might make future dev easier, for example a `hide_headers()` method to complement the `hide_index` method.